CONTRIBUTING.txt

```diff
@@ -379,6 +379,27 @@ To rebuild a full clean documentation, run:
     make clean
     make html
 
+Gallery
+~~~~~~~
+
+If you are contributing an example to the
+`gallery <https://scikit-image.org/docs/dev/auto_examples/>`_ (or editing an
```

Comment:
Should this be an internal link (ref?) instead? There is another link to a https://scikit-image.org/ page elsewhere (line 83), so I left it this way...

Reply:
I think a ref is better, good suggestion! Anyone else @scikit-image/core got an opinion?

Reply:
Done as part of bdfbbd24.

Then I thought I might as well change the other https://scikit-image.org/ link: 0fd5eb9.

Also, from https://thomas-cokelaer.info/tutorials/sphinx/rest_syntax.html, it seems that the link text at
https://github.com/scikit-image/scikit-image/blob/7e4840bd9439d1dfb6beaf549998452c99f97fdd/CONTRIBUTING.txt#L256
is unnecessary, 'Coordinate conventions' being the title following the `numpy-images-coordinate-conventions` label (file `doc/source/user_guide/numpy_images.rst`).